src/index.html

```diff
@@ -7,12 +7,61 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="picture product"
+      />
+      <div class="product card__prod">
+        <h3 class="product__title">
+          APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h3>
+        <p class="product__code">Product code: 195434</p>
+      </div>
+      <div class="rewiews card__rewie">
```

Comment:
There is a typo in the class name 'rewiews'. It should be 'reviews' to maintain consistency and avoid potential issues with styling or JavaScript functionality.